Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md

```diff
@@ -68,10 +70,44 @@ If you are using IISExpress (as with Visual Studio) you’ll need to add a stati
     </location>
   </configuration>
 ```
+### Forms
+To clarify we are not storing form submissions into Azure storage but the JSON files used to map the layout of the forms such as the templates used and workflows. Forms submissions are stored in the database.
+
+To configure Umbraco forms data its very similar to how media is setup above. You just need to provide another provider as a sibling to the media provider. In the snippet below I have added a provider with the alias of forms along side with the media provider.
+
+```xml
```

Comment:
I am not entirely sure I understand this code snippet here.
There is ONLY a provider for the forms here, right?
Looks like the media provider is closed in the same line as it's started.

I might be wrong though 🤔
Perhaps you could help me understand exactly what you're doing here?

Reply:
To keep the snippet short I removed all the xml code inside the media provider to just show the bare bones of it along side the forms provider.

Reply:
Alright! Makes sense 💪 
Perhaps that could be explained in [a note](https://our.umbraco.com/documentation/Contribute/Markdown-Conventions/#adding-notes-warnings-tips) before or after the snippet?
What do you think?